<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choseWaveToExtract</class>
 <widget class="QDialog" name="choseWaveToExtract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chose wavelength to extract</string>
  </property>
  <widget class="QTableWidget" name="tableOptions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Options</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableChoises">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>0</y>
     <width>151</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Choises</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pbAdd">
      <property name="text">
       <string>Add       </string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/go-next.png</normaloff>:/new/icons/imagenInte/go-next.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pbRemove">
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/go-previous.png</normaloff>:/new/icons/imagenInte/go-previous.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbAddAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add all</string>
   </property>
   <property name="icon">
    <iconset resource="lstImgs.qrc">
     <normaloff>:/new/icons/imagenInte/go-last.png</normaloff>:/new/icons/imagenInte/go-last.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRemoveAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove all</string>
   </property>
   <property name="icon">
    <iconset resource="lstImgs.qrc">
     <normaloff>:/new/icons/imagenInte/go-first.png</normaloff>:/new/icons/imagenInte/go-first.png</iconset>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxStep">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>20</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="lstImgs.qrc"/>
 </resources>
 <connections/>
</ui>
